dht-node/src/index.ts

```diff
@@ -21,14 +22,23 @@ async function main() {
     logger.fatal('Fatal: Database Version incorrect')
     throw new Error('Fatal: Database Version incorrect')
   }
-
-  // eslint-disable-next-line no-console
-  console.log(
-    `starting Federation on ${CONFIG.FEDERATION_DHT_TOPIC} ${
-      CONFIG.FEDERATION_DHT_SEED ? 'with seed...' : 'without seed...'
-    }`,
-  )
-  await startDHT(CONFIG.FEDERATION_DHT_TOPIC)
+  // first read from .env.devop if exist
+  let dhttopic = DEVOP.FEDERATION_DHT_TOPIC
+  logger.debug(`dhttopic set by DEVOP.FEDERATION_DHT_TOPIC=${DEVOP.FEDERATION_DHT_TOPIC}`)
+  if (!dhttopic) {
+    dhttopic = CONFIG.FEDERATION_DHT_TOPIC
+    logger.debug(
+      `dhttopic overwritten by CONFIG.FEDERATION_DHT_TOPIC=${CONFIG.FEDERATION_DHT_TOPIC}`,
+    )
+  }
+  let dhtseed = DEVOP.FEDERATION_DHT_SEED
+  logger.debug(`dhtseed set by DEVOP.FEDERATION_DHT_SEED=${DEVOP.FEDERATION_DHT_SEED}`)
+  if (!dhtseed) {
+    dhtseed = CONFIG.FEDERATION_DHT_SEED
+    logger.debug(`dhtseed overwritten by CONFIG.FEDERATION_DHT_SEED=${CONFIG.FEDERATION_DHT_SEED}`)
+  }
+  logger.info(`starting Federation on ${dhttopic} ${dhtseed ? 'with seed...' : 'without seed...'}`)
+  await startDHT(dhttopic)
```

Comment:
see above - lots of duplicate code which could be avoided when doing it via deploy script

Reply:
perhaps, but this is not in focus of this PR